pkg/depsdev/v3/api_test.go

```diff
@@ -21,13 +21,40 @@ import (
 
 	def "github.com/edoardottt/depsdev/pkg/depsdev/definitions"
 	"github.com/edoardottt/depsdev/pkg/depsdev/v3"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 var (
 	api = depsdev.NewV3API()
 )
 
+func TestGetProject(t *testing.T) {
```

Comment:
Unreliable. If someone stars the repo or fork it the test will fail

Reply:
Make sense, I've updated the test to skip comparing actual value.